src/client/components/page-header/page-header.vue

```diff
@@ -1,16 +1,87 @@
 <template>
-  <header class="page-header">
+  <header class="page-header" :class="image ? 'page-header--with-cover-image' : null" :style="image ? 'background-image:url(' + image.url + ');' : null">
```

Comment:
do you think we should add a progressive image loading technique for the background image? like [this](https://css-tricks.com/the-blur-up-technique-for-loading-background-images/)?